build-system/scripts/query_manifest

```diff
@@ -11,18 +11,44 @@ if [ $(jq "has(\"$REPO\")" $MANIFEST) == "false" ]; then
   exit 1
 fi
 
-function addRebuildPatterns {
-    local TYPE=$(jq -r ".\"$1\".rebuildPatterns | type" $MANIFEST)
-    if [ "$TYPE" == "string" ]; then
-      local FILE=$(jq -r ".\"$1\".rebuildPatterns" $MANIFEST)
-      local BUILD_DIR=$($0 buildDir $1)
-      PATTERNS=(${PATTERNS[@]} $(cat $BUILD_DIR/$FILE))
-    elif [ "$TYPE" == "array" ]; then
-      PATTERNS=(${PATTERNS[@]} $(jq -r ".\"$1\".rebuildPatterns | .[]" $MANIFEST))
-    else
-      >&2 echo "Missing rebuildPatterns property. Either filename as string, or patterns as array."
+function get_deps {
+  local TYPE=$(jq -r ".\"$1\".dependencies | type" $MANIFEST)
+  if [ "$TYPE" == "string" ]; then
```

Comment:
Thoughts on this logic being in JS in the future?

Reply:
I'd ask, why?

Reply:
More scrutable scripting language for most our devs - jq is its own DSL, basically

Reply:
Meh. Work for works sake. With chatgpt this took me 5 minutes and 1 line of jq. Glorious!